view_conversations/conv-viewer-webapi/src/db_types.ts

```diff
@@ -154,17 +155,19 @@ function addHandler(inputHandlers : HandlerSet, handler : SpannerFieldHandler<Da
   return inputHandlers;
 }
 const handlerSet = handlers.reduce<HandlerSet>(addHandler, {});
+const ScoreSubstrings = 'RockV6_1|Smirnoff_2';
 
 export function parseOutputRows<T>(rows: spanner.ResultRow[]) : T[] {
   const output : ParsedOutput[] = []
   for (const row of rows) {
     const ret:  { [fieldName:string] : string | string[] | Date | number | null } = {};
     for (const field of row) {
-      if(!(field.name in handlerSet)) {
+      const testname = new RegExp(ScoreSubstrings).test(field.name) ? 'Score' : field.name;
```

Comment:
Using string constants that have to match up is very fragile (partial renamings cause bugs that are hard to find). Can you make 'Score' into a global constant variable?